packages/devtools_test/lib/src/mocks/generated.dart

```diff
@@ -4,10 +4,15 @@
 
 // ignore_for_file: implementation_imports
 
-import 'package:devtools_app/src/service/vm_service_wrapper.dart';
+import 'package:devtools_app/devtools_app.dart';
```

Comment:
import the files you need directly instead of importing devtools_app.dart

Reply:
We imported devtools_app in mocks.dart. Why should we do it differently here?

Reply:
Ah, I realize now this file is part of a different package `devtools_test`. Leave as is then. If this file were part of `devtools_app/` it would be more clear to import the files directly instead of importing the file that exports the files (which is the situation in my comment here: https://github.com/flutter/devtools/pull/3982/files#r854386007)